src/txns/helium.c

```diff
@@ -95,7 +112,7 @@ void sign_tx(uint8_t *dst, uint32_t account, const uint8_t *tx, uint16_t length)
 	cx_ecfp_private_key_t privateKey;
     derive_helium_public_key(account, &privateKey, NULL);
 	cx_eddsa_sign(&privateKey, CX_RND_RFC6979 | CX_LAST, CX_SHA512, tx, length, NULL, 0, dst, 64, NULL);
```

Comment:
You could change this to `cx_eddsa_sign_no_throw` and check for an error, otherwise `cx_eddsa_sign` could throw and `privateKey` would not be cleared.

Reply:
@yhql thanks! i just pushed up a commit to address that

Reply:
Thanks, wouldn't you also check `cx_eddsa_sign_no_throw`'s return value also ?